src/Components/Circular.js

```diff
@@ -0,0 +1,5 @@
+import './Circular.css';
+
+const Circle = () => '';
+
+export default Circle;
```

Comment:
- [ ] I think this component does not use in your project. Please, consider removing it, to not be confused 👍